lib/Base/System/ClientState.vala

```diff
@@ -73,7 +73,7 @@ public partial class Backend.Client : Object {
     while (server_iter.next ("v", out server_variant)) {
       try {
         var server = unpack_server (server_variant);
-        servers.add (server);
+        servers.add_server (server);
```

Comment:
It's mostly a stylistic thing, but if you've normally got a collection of servers in a variable called "servers" where you pass an object in a parameter called "server" then I wouldn't bother with the `_server` part of `add_server()`. It's already heavily implied, and only necessary if you want to add multiple object types and you can't do Java/C#-style "pick the right method signature based on parameter type" (which I think Vala can't do).

Unless it clashes with something in the base classes? But I can only see `add_item()` at the moment.

Reply:
Don't really know why I extended this to `add_server`...

Makes sense to keep it as `add`.